examples/binding-scala/js/build.sbt

```diff
@@ -0,0 +1,13 @@
+enablePlugins(ScalaJSPlugin)
```

Comment:
Do we really need an sbt subproject here? Wouldnt it be simpler if the build only had the root project, and the scala source code would live in `src/main/scala` instead of under `js/`?